### DISCUSSION TITLE
Timezone?

### DISCUSSION THREAD

**Discussion by kenge1967** [tomasrudh/analogclock#8] [Q&A]
Is there any posibility for setting timezone, I want two different clocks with different timezones.

**Comment by tomasrudh** [comment]
Hi, that's a good idea, I will add it to the wish list.

Thanks!
/Tomas

**Reply by tomasrudh** [reply]
I have published a version supporting time zones.

**Reply by kenge1967** [reply]
Very nice, thanx

**Comment by shadowpuck99** [comment]
@kenge1967 
just wondering - were you able to display multiple clocks or zones? if so, any tips or hints?

thank you.

**Reply by tomasrudh** [reply]
You can have two cards with clocks showing different time zones. Use the setting timezone, for example:
timezone: Europe/Stockholm

**Reply by shadowpuck99** [reply]
ok - i thought so.  but, when i try to add the second card it ends up either blank or disappearing on the dashboard.

i'm still very new to home assistant - so, i'm definitely learning.  but, i would assume i'd just do another custom card, but, that doesn't seem to work for me.  i'm sure i'm missing something....

**Reply by shadowpuck99** [reply]
ok - dumb user error....:)  Forgot to account for the space in Los_Angeles (ie, don't type it as Los Angeles).....

**Reply by tomasrudh** [reply]
You are probably missing something, a little hard to say what with this limited information. I have verified that it do work having multiple clocks.

**Reply by shadowpuck99** [reply]
yes, it was totally an error on my part.  i forgot how some of the timezones are actually listed.
and, it seems like what happens if i try to tell it to do something that causes an error (like timezone: America/Los Angeles instead of timezone: America/Los_Angeles) it just displays a blank card on the dashboard.

again - that's the default dashboard i'm using so i don't have it configured beyond how it comes out of the box, so to speak.  so, now i know what that means if that happens!

needless to say - i've learned a lot already and this is the best way to learn!

thank you again for the help and responses!
